src/jobsautoreport/report.py

```diff
@@ -246,6 +256,11 @@ def get_report(self, from_date: datetime, to_date: datetime) -> Report:
         presubmit_subsystem_and_e2e_jobs = [
             job for job in subsystem_and_e2e_jobs if job.type == JobType.PRESUBMIT.value
         ]
+        postsubmit_subsystem_and_e2e_jobs = [
```

Comment:
we don't have subsystem/e2e postsubmit jobs at all
I think we need to track all postsubmit jobs as they interest us not because of their costs but because of their publish action (at least usually they publish something)

Reply:
you have a lot more to change with the naming... (``postsubmit_subsystem_and_e2e_jobs->postsubmit_jobs``)

Reply:
My bad